### PR TITLE
 Stuart, 10.1016/S0140-6736(21)02718-5 #401 

### PR DESCRIPTION
**Pull request by zachandfox**
[Stuart, 10.1016/S0140-6736(21)02718-5 #401](https://github.com/hivdb/covid-drdb-payload/issues/401)

- I could not find information regarding the upper- and lower-limit cut-offs, nor the specific isolates used

### CODE REVIEW

**Comment by zachandfox** [comment]
@KaimingTao what is our rule around the addition of vaccine combinations? I wasn't sure if _adding a new combo to vaccines.csv_ or _writing **AZD1222 + mRNA** and **Unknown vaccine** in stuart22-vacc.csv_ was preferable

**Comment by zachandfox** [comment]
Not exactly sure how to resolve [this check failure](https://github.com/hivdb/covid-drdb-payload/runs/7049154699?check_suite_focus=true)

**Comment by KaimingTao** [comment]
> Stuart, 10.1016/S0140-6736(21)02718-5 #401

It's adding a new combo in the `vaccines.csv` file.

File: tables/ref_isolate_pairs/stuart22-pair.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ref_name,control_iso_name,iso_name
+Stuart22,Wuhan-Hu-1,Wuhan-Hu-1 Spike
```
Comment (KaimingTao) [inline tables/ref_isolate_pairs/stuart22-pair.csv:2]: @zachandfox I think the 2nd line is an error. Two reasones.

1. Wuhan-Hu-1 and Wuhan-Hu-1 Spike are the same, they don't have mutaions
2. Wuhan-Hu-1 is test on virus isolates and Wuhan-Hu-1 spike is test on pseudovirus, they can't be paired.

**Pull request merged**